stdlib/sql/from.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/semantic"
 	_ "github.com/lib/pq"
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
These imports will probably need to be refactored so they aren't here. That applies for all of the database drivers. The reason for this is because we sometimes want a driver to be available and sometimes we don't. When we include this library in our cloud offering, the sqlite3 connector needs to be gone because it's a security vulnerability and there's no use case for it in a cloud environment.

But for flux itself, I can see how sqlite3 would be really useful and so I like this change. But this import needs to only exist within the tests and within the `cmd/flux` package. It cannot be here.

Reply:
For the moment, leave the other database drivers there and move this import to the packages I mentioned.

Reply:
Will get this moved - didn’t occur to me, but makes complete sense

Reply:
@jsternberg apologies for the question, but you are essentially talking about having the ability to build different versions of Flux yes (one with sqlite, one without, for example)? If so, then simply moving this import somewhere else won't work - as the main sql code (example shown below) shows:  
```
        // Retrieve the row reader implementation for the driver.
	var newRowReader func(rows *sql.Rows) (execute.RowReader, error)
	switch spec.DriverName {
	case "mysql":
		newRowReader = NewMySQLRowReader
	case "sqlite3":
		newRowReader = NewSqliteRowReader
	case "postgres", "sqlmock":
		newRowReader = NewPostgresRowReader
	default:
		return nil, errors.Newf(codes.Invalid, "sql driver %s not supported", spec.DriverName)
	}
```
So there would need to be a way of uncoupling the driver-specific functionality from here yes? I am sure you guys have already solved this, and have some way of "modularising" functionality so that you can pick-and-choose connectors at compile time... I just can't see where you are already doing this.

I just want to make sure I understand the approach you are taking (or want to take) so I don't waste time submitting useless garbage changes that don't do what you want ;-)

Reply:
ok - moved import to cmd/flux/main.go (and tests), and removed from everywhere else. commenting it out results in:
```
> sql.from(driverName:"sqlite3",dataSourceName:"hack",query:"select * from b")
Result: _result
Error: sql: unknown driver "sqlite3" (forgotten import?)
```
no other side-effects. removing comments, and it all works again as expected. Reason I am not 100% sure about this is that there appears to be no other "3rd party" imports being handled like this - so either I'm being super-dense, or so far, no other connector falls under this same treatment?

Reply:
I created this issue anyway for ourselves since when you said that the mysql driver wasn't working, I did not feel like I could just say "no, it is definitely working" without manually checking.

Thanks for checking on that.

https://github.com/influxdata/flux/issues/2006